R/generate_simulated_data.R

```diff
@@ -579,7 +590,9 @@ generate_simulated_data <- function(r_in_weeks = # nolint
   example_data <- list(
     ww_data = ww_data,
     hosp_data = hosp_data,
-    hosp_data_eval = hosp_data_eval
+    hosp_data_eval = hosp_data_eval,
+    rt_site_data = r_site, # temp?
```

Comment:
@cbernalz This is where you are having the issue right, where when you call `example_data <- generate_simulated_data()` and then `example_data$rt` you get NULL? I would try the following: 
1. test everything within the generate simulated data function. Make sure `r_site` and `rt` are not NULL and are in fact a matrix and a vector respectively
2. try saving them as tibbles here (could also bind them to subpop spine and pivot them). I don't see why that would be the issue but am just trying to think what might be wrong here. 

Other thing is just to be absolutely sure you reinstalled the package before running the generate simulated data function but I am guessing you tried that already!

Reply:
Do you think it could be that reinstalling the package does not use the local version?  Because when I ran things just using the function and not with the package, I did get output for `rt_site_data` and `rt_global_data`.  When I don't do this and only use the package functions, I run each line of the `vignette_data.R`, `simulated_data$rt_global_data` and `simulated_data$rt_site_data` do not exist so this returns `NULL`.

Reply:
Hmm if you are using `remotes::install_local(force = TRUE)` I don't think so. Only thing I can think of is if you didn't reload the library (`library(wwinference)`)

Reply:
Just a question here, it seems that you were able to get all the package data saved. What was the fix?

Reply:
Was this resolved @cbernalz?